src/main/java/net/pms/configuration/PmsConfiguration.java

```diff
@@ -2720,12 +2722,29 @@ public void setIpFilter(String value) {
 		configuration.setProperty(KEY_IP_FILTER, value);
 	}
 
-	public void setPreventsSleep(boolean value) {
-		configuration.setProperty(KEY_PREVENTS_SLEEP, value);
-	}
-
-	public boolean isPreventsSleep() {
-		return getBoolean(KEY_PREVENTS_SLEEP, true);
+	public void setPreventSleep(PreventSleepMode value) {
+		if (value == null) {
+			throw new NullPointerException("value cannot be null");
+		}
+		configuration.setProperty(KEY_PREVENT_SLEEP, value.getValue());
+		PMS.get().getSleepManager().setMode(value);
+	}
+
+	public PreventSleepMode getPreventSleep() {
+		PreventSleepMode sleepMode = null;
+		String value = getString(KEY_PREVENT_SLEEP, null);
+		if (value == null && configuration.containsKey(KEY_OLD_PREVENTS_SLEEP)) {
+			// Backwards compatibility
+				sleepMode =
```

Comment:
@Sami32 The indentation issue depends on how you work I guess. I "refactor" the code quite a lot while I work with it as I discover that the code would be better organized another way. I then move blocks of code around, which can mean that the code ends up being at a "higher level of indentation" then it was when I wrote it, which - together with renaming the identifiers might mean that something that fit nicely on one line at one stage suddenly doesn't. I don't want to have to rewrite/change working code just because the line length changes because of such reorganizations.

CheckStyle has both Oracle's and Google's styles by default, neither fits very well with our current "style". I used one of them as a starting poing for the current CheckStyle configuration, but I don't remember which. 

I know very little about golang, but I did a little bit of reading now and it seems that the language is very limited (most even remotely "advanced" concepts are excluded). That makes it very easy to enforce a style 😉 

I'm not saying that there's a need for multi-line ternaries every day, but when the need is there I'd like to know how to format it - especially since I've had a hard time finding a format I'm "satisfied with". 

When it comes to preventing temporary variables, I'll try to give a short pseudo example:

```java
    type tempArg1;
    if (condition1) {
        tempArg1 = value1;
    } else {
        tempArg1 = value2;
    }

    type tempArg2;
    if (condition2) {
        tempArg2 = value3;
    } else {
        tempArg1 = value4;
    }

    method(tempArg1, tmpArg2);
```

Using ternaries this could be expressed much easier and cleaner:
```java
    method(condition1 ? value1 : value2, condition2 ? value3 : value4);
```

The above is using just two "conditional" arguments. As the number of "conditional" arguments grow, the difference grows exponentially.